src/app/client/src/app/modules/certificate/models/cert-config-model/cert-config-model.ts

```diff
@@ -57,8 +57,25 @@ export class CertConfigModel {
 
     }
 
-    prepareCreateAssetRequest(rawFormValues) {
+    prepareCreateAssetRequest(rawFormValues, channel, images) {
         console.log(rawFormValues);
+
+    const signatoryList = [{
+       'image': _.get(images, 'SIGN1.url'),
```

Comment:
User const mapping object to get
1. SignatoryLog( 1 & 2)
2. SignatoryText( 1 & 2)
3. Logo( 1 & 2)